src/tests/ftest/container/snapshot.py

```diff
@@ -426,8 +427,11 @@ def test_snapshots(self):
                           "still available", num_transactions)
 
         # (5)Verify the snapshots data
```

Comment:
suggest moving this single line  comment (5)Verify ... 
inside the loop, just like step (6), and indenting both comments with the code that's there.

Reply:
Comments are in the test description.